WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Reader Messages/PointOfSaleCardPresentPaymentPreparingForPaymentMessageView.swift

```diff
@@ -4,10 +4,7 @@ struct PointOfSaleCardPresentPaymentPreparingForPaymentMessageView: View {
     let viewModel: PointOfSaleCardPresentPaymentPreparingForPaymentMessageViewModel
 
     var body: some View {
-        let messageViewModel = POSCardPresentPaymentMessageViewModel(showProgress: true,
-                                                                     title: viewModel.title,
-                                                                     message: viewModel.message)
-        POSCardPresentPaymentMessageView(viewModel: messageViewModel, style: .dimmed)
+        PointOfSaleCardPresentPaymentValidatingOrderMessageView(title: viewModel.title, message: viewModel.message)
```

Comment:
This is named for one specific step, but being used for two steps.

Since the view is the same other than the strings shown that seems good, but the naming could be confusing. Perhaps we should rename it?

```suggestion
        PointOfSaleCardPresentPaymentActivityIndicatingMessageView(title: viewModel.title, message: viewModel.message)
```